src/main/resources/application.yaml

```diff
@@ -100,3 +100,12 @@ http:
       timeout: ${HTTP_CLIENT_CONNECTION_TIMEOUT:10000}
     read:
       timeout: ${HTTP_CLIENT_READ_TIMEOUT:30000}
+
+logging:
+  level:
+    uk:
+      gov:
+        hmcts:
+          reform:
+            ccd:
+              documentam: ${CCD_DOCUMENTAM_LOG_LEVEL:INFO}
```

Comment:
1 line is plenty.

Reply:
This is a yaml file and the regular formatting.

Reply:
ahh, didn't notice for a moment. :)